tests/utils.py

```diff
@@ -45,7 +45,9 @@
     return tuple(nodes_list)
 
 
-async def set_up_routers(router_confs):
+async def set_up_routers(router_confs=[0, 0]):
```

Comment:
Should we avoid the mutable default value here though it's unlikely to be changed? It seems we can simply change to a tuple `(0, 0)` to avoid this.

Reply:
yep, good catch